### ISSUE TITLE
Forgotton Indicator: Documentation

### ISSUE THREAD

**Issue by GeorgLink** [OSSHealth/HealthIndicators#5]
We know that documentation is essential to learning how to use and incorporate an open source software.

Contribution to documentation has been highlighted as valuable.
Documentation can be part of the code, readme files, wiki, website etc. 

Interestingly, quality documentation has not come up in the interviews, yet.
The CII considers the existence of a website, but that does not necessarily qualify as a documentation.

Did I overlook something?

**Comment by germonprez** [comment]
I don't think so. This simply hasn't come up in the interviews so far. And I'm not sure that I've seen it in the papers either.

**Comment by GeorgLink** [comment]
Crowston, Howison, & Annabi (2006) included **"Documentation quality"** in their "concepts for IS success in FLOSS context"

But I have not heard it come up anywhere else in the mean time.
